auto_install/install.sh

```diff
@@ -1085,13 +1085,44 @@ setVPNDefaultVars(){
 	fi
 }
 
+generateRandomSubnet() {
+	# Source: https://community.openvpn.net/openvpn/wiki/AvoidRoutingConflicts
+	declare -a SUBNET_EXCLUDE_LIST=(10.0.0.0/24 10.0.1.0/24 10.1.1.0/24 10.1.10.0/24 10.2.0.0/24 10.8.0.0/24 10.10.1.0/24 10.90.90.0/24 10.100.1.0/24 10.255.255.0/24)
+
+	readarray -t CURRENTLY_USED_SUBNETS <<< "$(ip -o addr show | grep -oE '[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\/[0-9]{1,2}')"
```

Comment:
Actually since the goal is to avoid routing conflicts, it makes more sense to get subnets for which there is a route in the routing table.